basics/Main.java

```diff
@@ -0,0 +1,79 @@
+import java.time.LocalDateTime;
+import java.util.Random;
+
+public class Main {
+  public static void main(String[] args) {
+    // Feature Pluralize
+    int dogCount = 1;
+    System.out.println("I own " + dogCount + " " + pluralize("dog", dogCount) + ".");
+
+    int catCount = 2;
+    System.out.println("I own " + catCount + " " + pluralize("cat", catCount) + ".");
+
+    int turtleCount = 0;
+    System.out.println("I own " + turtleCount + " " + pluralize("turtle", turtleCount) + ".");
+
+    // Feature Flipping Coins
+    flipNHeads(2);
+
+    // Feature Command Line Clock
+    clock();
+  }
+
+  // Feature Pluralize
+  public static String pluralize(String word, int count) {
+    String plural = "" + 's';
```

Comment:
Single quotes denotes a `char` while double quotes denotes a `String`. Instead of `String plural = "" + 's';` you can use double quotes `String plural = "s";`